src/output-json-alert.c

```diff
@@ -572,6 +573,68 @@ static void AlertAddFiles(const Packet *p, JsonBuilder *jb, const uint64_t tx_id
     }
 }
 
+bool PacketCheckAction(const Packet *p, const uint8_t a)
```

Comment:
in master 6 we have been fixing things differently, I think you should use `PACKET_TEST_ACTION` instead